README.md

```diff
@@ -13,7 +13,13 @@ The generated labels include:
 
 <img src="Example.svg">
 
-# Usage
+# Usage (Docker)
```

Comment:
I'd like to move this after the python based usage, as running it through python directly is the recommended and intended usage.

Reply:
ok fixed.